package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cylc-ui",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
I think I added this as I noticed the footer was showing 0.1, but can be moved to a separate PR if necessary.